doc/src/network-programming-guide.md

```diff
@@ -74,7 +76,8 @@ gotIp(IpInfo) ->
     io:format("Got IP: ~p~n", [IpInfo]).
 
 disconnected() ->
-    io:format("Disconnected from AP.~n").
+    io:format("Disconnected from AP, attempting to reconnect~n"),
+    network:connect().
```

Comment:
do we need a delay here eg 500-1000ms or something? (ideally exponential backoff - but static delay should be fine..)

Reply:
No. This just signals the driver to start attempting a connection, if that attempt succeeds the connected callback will be triggered, otherwise a disconnected callback will be triggered - in which case the attempt will be made again… this would function exactly as the driver always has… by defining a custom callback the user could and any extra delay, or scan for networks instead.

Reply:
The only difference is this was connection was called internally, so stop would never be possible. Now by defining a callback the user can control when a reconnection (or now a new connection).